formatters/simplecov/simplecov.go

```diff
@@ -1,17 +1,17 @@
 package simplecov
 
 import (
-	"encoding/json"
 	"os"
 	"strings"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/codeclimate/test-reporter/env"
 	"github.com/codeclimate/test-reporter/formatters"
 	"github.com/pkg/errors"
 )
 
-var searchPaths = []string{"coverage/.resultset.json"}
+var simpleCovJsonFormatterPath, legacyPath string = "coverage/coverage.json", "coverage/.resultset.json"
+
+var searchPaths = []string{simpleCovJsonFormatterPath, legacyPath}
```

Comment:
Can this be inlined? Doesn't look like we use those vars anywhere else.

Reply:
Done 👍🏼